### PR TITLE
Adding the symfony 6 compatibility

### PR DESCRIPTION
**Pull request by vincent-joignie-dd**
Codeception 5 now support Symfony 6.

Using robo-paracept with Symfony 6 is leading to:
```codeception/robo-paracept 2.0.0 requires symfony/finder >=2.7 <6.0 -> found symfony/finder[v2.7.0, ..., v2.8.52, v3.0.0, ..., v3.4.47, v4.0.0, ..., v4.4.44, v5.0.0, ..., v5.4.11] but the package is fixed to v6.0.11 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.```

during the `composer install`.

I'm trying to make the robo-paracept library compatible with Symfony 6.

### CODE REVIEW

**Comment by vincent-joignie-dd** [comment]
@DavertMik A little help around here pls?

**Comment by akalineskou** [comment]
I've started experimenting with this...
These are the changes I've needed for now

`composer.json`
```diff
- "consolidation/robo": "3.0.*",
+ "consolidation/robo": "3.0.* | 4.0.*",
- "symfony/finder": ">=2.7 <6.0",
+ "symfony/finder": ">=2.7",
```

`src/Splitter/TestsSplitter.php`
```diff
// autoload PHPUnit files
- \Codeception\PHPUnit\Init::init();
+ // \Codeception\PHPUnit\Init::init();
```

I'm not sure why the `Init::init()` was being called, since I can't find it anywhere in the codeception repo

**Comment by denkurbatov** [comment]
Having same issue with Laravel 9.22.1

**Comment by route33** [comment]
Having the same issue here. @akalineskou I see your changes https://github.com/Codeception/robo-paracept/compare/Codeception:master...akalineskou:master?expand=1
Did it work?

**Comment by akalineskou** [comment]
@route33 Yes, it works with those changes for my case

**Comment by vansari** [comment]
Hi,

I will check the PR.

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "require": {
         "php": "^7.4 | ^8.0",
         "consolidation/robo": "3.0.*",
```
Comment (vansari) [inline composer.json:18]:
```suggestion
        "consolidation/robo": "3.0.* | 4.0.*",
```
For symfony 6 you need also robo 4.0.* :)

**Review by vansari — CHANGES_REQUESTED** [review]

**Comment by vansari** [comment]
@DavertMik & @Arhell 
I will create a new Release Version 3.0 because of breaking changes if we update deps of symfony/finder to ^6.0 and consolidation/robo to ^4.0. 

@vincent-joignie-dd FYI 

I will also check the Tests deps

**Comment by vansari** [comment]
@vincent-joignie-dd @akalineskou @denkurbatov @route33 Version 3.0 with PHP 8.1 and robo 4.0 and codeception 5 support is out now

**Comment by route33** [comment]
@vansari this is not good enough. I use PHP 8.0 and for some segmentation fault reasons cannot upgrade to 8.1.
Robo paracept cannot be used in PHP 8.0 along with codeception/codeception: ^5.0 and symfony 6.0.*

```bash
Cannot use codeception/robo-paracept's latest version 3.0.0 as it requires php ^8.1 which is not satisfied by your platform.
Using version ^2.0 for codeception/robo-paracept
./composer.json has been updated
Running composer update codeception/robo-paracept
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires codeception/robo-paracept ^2.0 -> satisfiable by codeception/robo-paracept[2.0.0, 2.0.1, 2.0.x-dev].
    - codeception/robo-paracept[2.0.0, ..., 2.0.x-dev] require symfony/finder >=2.7 <6.0 -> found symfony/finder[v2.7.0-BETA1, ..., 2.8.x-dev, v3.0.0-BETA1, ..., 3.4.x-dev, v4.0.0-BETA1, ..., 4.4.x-dev, v5.0.0-BETA1, ..., 5.4.x-dev] but the package is fixed to v6.0.11 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

**Comment by vansari** [comment]
I will check it again 😊

**Comment by vansari** [comment]
@route33 please checkout the new 2.1 version
https://github.com/Codeception/robo-paracept/releases/tag/2.1.0

If it is possible please open a new issue for new questions or bugs

**Comment by akalineskou** [comment]
Worked for me with the 2.1.0 release!

**Pull request merged**